enhancements/cluster-logging/cluster-logging-log-forwarding.md

```diff
@@ -0,0 +1,309 @@
+---
+title: cluster-logging-log-forwarding
+authors:
+  - "@jcantrill"
+reviewers:
+  - "@bparees"
+  - "@ewolinetz"
+  - "@jeremyeder"
+approvers:
+  - TBD
+creation-date: 2019-09-17
+last-updated: 2019-09-17
+status: implementable
+see-also:[]
+replaces:[]
+superseded-by:[]
+---
+
+# cluster-logging-log-forwarding
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Summary
+
+The purpose of log forwarding is to provide a declarative way by which adopters of cluster logging can ship
+container and node logs to destinations that are not necessarily managed by the OKD cluster logging infrastructure. This is on-cluster endpoints like the Elasticsearch cluster deployed by cluster logging, an organization's Kafka message bus, or off-cluster to existing log aggregation solutions (e.g. Splunk, Graylog).  Additionally, log forwarding enables administrators to deploy only the components needed to ship the logs without needing to deploy the entire cluster logging stack.
```

Comment:
Nit, but can't we already deploy just components to ship logs today? We don't need to deploy the entire stack today...

Reply:
We can but I think its not as obvious: deploy fluent, make unmanaged, edit config.  I'll remove it

Reply:
This may actually be a current functionality gap, but we don't watch the contents of fluentd -- so the operator doesn't need to go unmanaged.

Reply:
It's a gap IMO.  Part of the initial PR I will be submitting hashes the config, adds it to fluent so that the config is reverted should someone manually change it.  The deployment of fluent should be consistent from the operators perspective.